entities/entities/money_printer/init.lua

```diff
@@ -1,27 +1,50 @@
--- RRPX Money Printer reworked for DarkRP by philxyz
 AddCSLuaFile("cl_init.lua")
 AddCSLuaFile("shared.lua")
 include("shared.lua")
 
-ENT.SeizeReward = 950
+local function PrintMore(ent)
+    if not IsValid(ent) then return end
+
+    ent.sparking = true
+    timer.Simple(3, function()
+        if not IsValid(ent) then return end
+        ent:CreateMoneybag()
+    end)
+end
+
+function ENT:StartSound()
+    self.sound = CreateSound(self, Sound("ambient/levels/labs/equipment_printer_loop1.wav"))
+    self.sound:SetSoundLevel(52)
+    self.sound:PlayEx(1, 100)
+end
+
+function ENT:initVars()
```

Comment:
Pretty much duplicate code with shared.lua:10 onwards. Why isn't this function shared?